cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -6256,6 +6256,12 @@ declare namespace Deno {
     info: ServeHandlerInfo,
   ) => Response | Promise<Response>;
 
+  export interface Serve {
+    fetch: (
+      request: Request,
+    ) => Response | Promise<Response>;
+  }
```

Comment:
Seems confusing that `Deno.Serve` and `Deno.serve` are unrelated.

Reply:
Happy to hear suggestions for other names

Reply:
ServeDefaultExport